src/client/datascience/jupyter/kernels/kernelExecution.ts

```diff
@@ -54,12 +54,10 @@ export class KernelExecution implements IDisposable {
         );
     }
 
-    @captureTelemetry(Telemetry.ExecuteNativeCell, undefined, true)
     public async executeCell(notebookPromise: Promise<INotebook>, cell: NotebookCell): Promise<NotebookCellRunState> {
         if (cell.kind == NotebookCellKind.Markup) {
             return NotebookCellRunState.Success;
         }
-        sendKernelTelemetryEvent(cell.notebook.uri, Telemetry.ExecuteNativeCell);
```

Comment:
This is redundant, as this code gets called by interactive as well.
We have an event `ExecuteCell` which gets called higher up, for interactive & native.